.circleci/config.yml

```diff
@@ -10,9 +10,7 @@ jobs:
       - checkout
 
       - restore_cache:
-          keys:
-          - v1-dependencies-{{ checksum "package.json" }}
-          - v1-dependencies-
+          key: v1-dependencies-{{ checksum "package.json" }}
```

Comment:
The second one is a fallback [if the checksum fails](https://circleci.com/docs/2.0/caching/#example-caching-configuration).